mp4parse_capi/src/lib.rs

```diff
@@ -248,7 +248,7 @@ impl Read for mp4parse_io {
 
 /// Allocate an `mp4parse_parser*` to read from the supplied `mp4parse_io`.
 #[no_mangle]
-pub unsafe extern fn mp4parse_new(io: *const mp4parse_io) -> *mut mp4parse_parser {
+pub unsafe extern fn mp4parse_new(io: *const mp4parse_io, log: bool) -> *mut mp4parse_parser {
```

Comment:
Unless we need logging specifically withing mp4parse_new() I'd rather mirror the rust api and have a separate `mp4parse_log(parser: *mut mp4parse_parser, enable: bool)` call to enable debug output, so we don't have to add the `false` every time we invoke it.

If you want to write a log adaptor which would let us forward trace messages to gecko's logging that would be helpful though. Could be an extra callback in mp4parse_io, or a new method to dump a log buffer.